Server/Test_Plan_OCIS_Smoke_Test.md

```diff
@@ -98,15 +98,15 @@ Marie:
 - [ ] can see space without accepting
 - [ ] change image and description
 - [ ] can upload file/folder and create file and folder
-- [ ] cannot create resource sharing and space sharing
+- [ ] can create/share resource but cannot share space
```

Comment:
can create resource but cannot share resource and share space

Reply:
<img width="955" alt="Screenshot 2022-11-11 at 09 09 36" src="https://user-images.githubusercontent.com/84779829/201295765-14250036-8c67-4a57-9cad-411f828ec4aa.png">

Reply:
may be i can update it by `can create resource but cannot share space or space resource`

Reply:
yes, sure.